Formula/lensfun.rb

```diff
@@ -30,12 +29,6 @@ class Lensfun < Formula
   depends_on "libpng"
   depends_on "python@3.9"
 
-  # This patch can be removed when new Lensfun release (v0.3.96) is available.
-  patch do
-    url "https://github.com/lensfun/lensfun/commit/de954c952929316ea2ad0f6f1e336d9d8164ace0.patch?full_index=1"
-    sha256 "67f0d2f33160bb1ab2b4d1e0465ad5967dbd8f8e3ba1d231b5534ec641014e3b"
-  end
-
   def install
     system "cmake", ".", *std_cmake_args
     system "make", "install"
```

Comment:
This is hitting Python 3.9+ setuptools/distutils issue. It will need a workaround similar to `git-cola`:

https://github.com/Homebrew/homebrew-core/blob/5f4f021c456b71aa915f6df70ef7ab4f2e736e84/Formula/git-cola.rb#L25-L27

Reply:
Thanks a lot @cho-m, that did the trick!